browser/profiles/profile_util.cc

```diff
@@ -170,20 +169,9 @@ bool IsSessionProfilePath(const base::FilePath& path) {
          base::FilePath(FILE_PATH_LITERAL("session_profiles"));
 }
 
-bool IsTorProfilePath(const base::FilePath& path) {
-#if BUILDFLAG(ENABLE_TOR)
-  return IsSessionProfilePath(path) &&
-         path.BaseName() == base::FilePath(tor::kTorProfileDir);
-#else
-  return false;
-#endif
-}
-
 bool IsTorProfile(content::BrowserContext* context) {
 #if BUILDFLAG(ENABLE_TOR)
-  DCHECK(context);
-  return IsTorProfilePath(
-      Profile::FromBrowserContext(context)->GetOriginalProfile()->GetPath());
+  return context->IsTor();
```

Comment:
Now we could access it through BrowserContext directly, we could probably eliminate this function and let callers use it through the context directly to eliminate their dependency on browser/profiles.

Reply:
fixed in eb3dc9639c4f34fe6ab94d4c060108922428cb13